app/controllers/grocery_lists_controller.rb

```diff
@@ -0,0 +1,53 @@
+class GroceryListsController < ApplicationController
+  include GroceryListsHelper
+  before_action :find_list, only: [:show, :edit, :update, :destroy, :add_item, :delete_item]
+
+  def find_list
+    @list = GroceryList.find(params[:id])
+  end
+
+  def index
+    @lists = GroceryList.all
+  end
+
+  def new
+    @list = GroceryList.new
+  end
+
+  def show
+  end
+
+  def create
+    @list = GroceryList.new(list_params)
+    @list.save
```

Comment:
`.save` returns a true or false if the save was successful. It looks like you're not handling the failure case? You could flash and error and redirect them to new form page again.